src/shaders.rs

```diff
@@ -123,52 +92,48 @@ pub fn full_shaders(
                 },
             )?
         }};
-        ($name:ident, $bindings:expr, $defines:expr, $cpu:expr) => {{
-            add_shader!($name, stringify!($name), $bindings, &$defines, $cpu)
+        ($name:ident, $bindings:expr, $cpu:expr) => {{
+            add_shader!($name, stringify!($name), $bindings, $cpu)
         }};
-        ($name:ident, $bindings:expr, $defines:expr) => {
+        ($name:ident, $bindings:expr) => {
             add_shader!(
                 $name,
                 $bindings,
-                &$defines,
-                CpuShaderType::Present(cpu_shader::$name)
+                CpuShaderType::Present(vello_shaders::cpu::$name)
             )
         };
         ($name:ident, $bindings:expr) => {
-            add_shader!($name, $bindings, &full_config)
+            add_shader!($name, $bindings)
```

Comment:
This is the exact as above and can be removed

Reply:
Pretty interesting, does rust not cover equal macro branches? O.o

Reply:
Good catch!